src/charm.py

```diff
@@ -297,14 +297,19 @@ def _prepare_configs(self, container: Container) -> bool:
         """
         if container.exists(MYSQLD_CONFIG_FILE):
             return True
-        try:
-            (
-                innodb_buffer_pool_size,
-                innodb_buffer_pool_chunk_size,
-            ) = self._mysql.get_innodb_buffer_pool_parameters()
-        except MySQLGetInnoDBBufferPoolParametersError:
-            self.unit.status = BlockedStatus("Error computing innodb_buffer_pool_size")
-            return False
+
+        if profile == "testing":
+            innodb_buffer_pool_size = 20971520
+            innodb_buffer_pool_chunk_size = 1048576
```

Comment:
no max_connections for K8s (comparing to VM)? 0_o

Reply:
is in the pipeline (dpe-2069) - this will unblock it